framework/test/mocha/functional/http/get/dapps.js

```diff
@@ -319,20 +386,20 @@ describe('GET /dapps', () => {
 		});
 
 		describe('unknown=', () => {
-			it('using empty string should return all results', async () => {
-				return dappsEndpoint.makeRequest({ unknown: '' }, 200).then(res => {
-					expect(res.body.data).to.have.length.at.least(registeredDappsAmount);
-				});
+			it('using empty string should return UNKNOWN_PARAM error', async () => {
```

Comment:
I think we may want to move this change to 1.7 instead of 1.6 I understand the fix and reasoning behind it but it's a behaviour change.  (Same for below)

@MaciejBaj @diego-G thoughts?

Reply:
If I have to revert the behavior so it returns all results when you pass an invalid filter to the endpoint, then I'll have to remove the `validateFilters` and `validateOptions` functions for this `/dapps` endpoint. This means there is no validation for all other fields as well (old bug ->). Some sort option like `id:wrong` would throw `unexpected error`.

@lsilvs thoughts?

Reply:
This behaviour was a bug and this test shouldn't cover this bug imo.
It changes the behaviour but fixing it (which is fine for me)

Reply:
Resolving as there's agreement that the change is acceptable.